backend/Dockerfile.prod

```diff
@@ -28,4 +28,4 @@ COPY --from=build  /home/gradle/graphql-api/backend/insights.jar insights.jar
 RUN addgroup -S appgroup && adduser -S appuser -G appgroup
 USER appuser
 
-ENTRYPOINT ["java", "-javaagent:insights.jar", "-jar", "app.jar"]
+ENTRYPOINT ["java", "-javaagent:insights.jar", "-XX:MaxRAMPercentage=75", "-XX:InitialRAMPercentage=75", "-jar", "app.jar"]
```

Comment:
with` XX:InitialRAMPercentage `setting the minimum and `XX:MaxRAMPercentage` setting the maximum, whats the advantage of the both being set to the same percentage? would setting the minimum to a lower value allow the jvm to scale down the memory usage of it doesn't need it?

Reply:
If we do need to grow/shrink the heap in demand to a particular load then we may run into GC pauses that affect response times. I don't think it's a huge concern, but the thought here was why not just claim the entire heap from the start?